lib/BaseDownloadController.js

```diff
@@ -62,18 +62,21 @@ module.exports = class BaseDownloadController extends CompositeController {
         objectPath.set(reqOpt, 'headers.If-Modified-Since', objectPath.get(imsObj, 'last-modified'));
       }
 
-      try {
-        reqOpt = await this._fetchHeaderSecrets(reqOpt);
-      } catch (e) {
-        // error fetching header secrets
-        if (optional) {
-          ++optionalResourceFailure;
-          this.log.warn(e.message);
-          this.updateRazeeLogs('warn', { controller: 'BaseDownload', warn: e.message, url: url });
-          this.log.debug(`skipping download for ${url}`);
-          continue; // shouldnt continue to try to download if unable to get secret headers
-        } else {
-          return Promise.reject(e.message);
+      const git = objectPath.get(request, 'options.git');
+      if (!git) {
```

Comment:
why dont we want header secrets fetched? i thought the reason we added `headersFrom` was for gitops?

Reply:
I am fetching them in my overload added function b/c I need them to get the urls

Reply:
see remoteresource repo comment, i dont think this is necessary

Reply:
i dont love that we are putting git specific logic in the baseDownloadController. Is there any harm in letting the fetchHeaderSecrets run again? or another non-git specific way to skip it?

Reply:
I mean it still works if fetchHeaderSecrets runs again, so I can just remove the git check then